src/sentry/testutils/pytest/sentry.py

```diff
@@ -50,11 +50,14 @@ def configure_split_db() -> None:
     settings.DATABASE_ROUTERS = ("sentry.db.router.SiloRouter",)
 
 
-DEFAULT_SILO_MODE_FOR_TEST_CASES = SiloMode.MONOLITH
+DEFAULT_SILO_MODE_FOR_TEST_CASES = SiloMode.REGION
```

Comment:
This should remain `SiloMode.MONOLITH` until the rest of the cases are fixed.

Reply:
Yep, that makes sense. I just set it for now to allow us to get a new test run.